app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -20,19 +21,20 @@ type Section = Item & {
 type Props = {
   sections: Section[],
   activeSection: ?string,
-  activeItem: ?string
+  activeItem: ?string,
+  currentApi: ?string
```

Comment:
This should be of type `Api`:
```suggestion
  currentApi: ?Api
```
It can be imported from the types folder like this:
`import type { Api } from 'Types'`